src/pages/MonthlyGood/MonthlyGoodSetupPage.vue

```diff
@@ -540,6 +542,15 @@ export default {
 
 		// Sanitize and set initial form values.
 		// Initial group from prop
+		const hasLoggedInResult = this.apollo.readQuery({
+			query: hasEverLoggedInQuery
+		});
+		if (!hasLoggedInResult?.hasEverLoggedIn) {
```

Comment:
There shouldn't be an if statement around this because the COVID-19 category should be removed as an option for everybody.